package.json

```diff
@@ -1,16 +1,27 @@
 {
     "name": "cardknox-sdk-js",
     "version": "1.0.2201.1601",
-    "description": "",
+    "description": "A library for communicating with Cardknox supported terminals and the Cardknox Gateway",
     "main": "dist/index.min.js",
+    "license": "",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/cardknox/cardknox-sdk-js.git"
+    },
+    "author": {
+        "email": "info@cardknox.com"
```

Comment:
should we make this developersupport@cardknox as well?

Reply:
Ok,
I just copied from React component